frontend/src/pages/LandingPage/LandingPage.jsx

```
@@ -0,0 +1,676 @@
+            title="What Our Users Say"
+            subtitle="Discover how Geeky Chef is revolutionizing cooking experiences for food enthusiasts, health-conscious individuals, and busy people"
+          />
+
+          <SimpleGrid columns={{ base: 1, md: 2 }} gap={8} mt={10}>
+            <TestimonialCard
+              name="Emma Thompson"
+              role="Food Blogger"
+              quote="Geeky Chef has completely transformed how I develop recipes. The AI image analysis gives me precise nutritional information instantly, and the meal planning feature helps me create content for my followers efficiently."
+              avatar="/api/placeholder/100/100"
+              delay={0.1}
+            />
```

Biome (1.9.4)

[error] Enforce that elements with ARIA roles must use a valid, non-abstract ARIA role.

Check WAI-ARIA for valid roles or provide options accordingly.
Unsafe fix: Remove the invalid role attribute.
 Check the list of all valid role attributes.


(lint/a11y/useValidAriaRole)